spec/lib/tasks/fix_student_loan_repayment_amounts_spec.rb

```diff
@@ -3,6 +3,19 @@
 Rails.application.load_tasks
 
 describe "fix_tslr_student_loan_amounts" do
+  around do |example|
+    original_stdout = $stdout
+    original_stderr = $stderr
+
+    $stdout = File.open(File::NULL, "w")
+    $stderr = File.open(File::NULL, "w")
+
+    example.run
+
+    $stdout = original_stdout
+    $stderr = original_stderr
+  end
```

Comment:
i want to slow tidy stdout when rspec runs

Reply:
Should probably just delete this spec and the rake task as it seems to be a one off task https://dfedigital.atlassian.net.mcas.ms/browse/CAPT-1877